infrastructure/code/template/details.html

```diff
@@ -39,16 +39,6 @@
       background: #595a5c;
     }
 
-    html,
```

Comment:
These got moved to Intellij in https://github.com/snyk/snyk-intellij-plugin/pull/541 because in VSCode we have different styling as seen in https://github.com/snyk/vscode-extension/pull/467